Minimum-Viable-Data-Space.md

```diff
@@ -0,0 +1,47 @@
+# What is a Minimum Viable Data Space? 
+A minimum viable data space (MVDS) is a combination of components to initiate a data space with just enough features to be usable for secure and sovereign data exchange. 
+It aims to facilitate the work of experimenters by shortening the implementation time (by avoiding lengthy details that would slow down the first release). 
+This allows them to start with a first working version (where secure and sovereign data exchange is granted), where the development team can iterate, identify and respond to the assumptions about the requirements of the data space. 
+
+A Minimum Viable Data Space consists of the items that are available in Portfolio of IDS-compliant Building Blocks that are currently available 
+on the [Build Components page](https://github.com/International-Data-Spaces-Association/idsa/blob/main/how-to-build-data-spaces/3-Build-Components.md) of the 
+[How to Build Data Spaces?](https://github.com/International-Data-Spaces-Association/idsa/tree/main/how-to-build-data-spaces) section. 
+
+MVDS is one of the IDS Deployment Scenarios provided by IDSA Head Office. 
+
+(*The IDS Deployment Scenarios is currently under development and will be available by Q2, 2022*).
+
```

Comment:
The deployment scanrios are your best practice or inspiration how to build data spaces:
- what were your requirements?
- What did you add?
- ...